base/some.jl

```diff
@@ -33,47 +33,39 @@ function show(io::IO, x::Some)
 end
 
 """
-    coalesce(x, y...)
+    notnothing(x)
```

Comment:
As you noted, I think we can just remove `notnothing` and use `something` instead. I don't think we use it in situations where accidentally unwrapping `Some` is possible. On the contrary, I only used `notnothing` in places where `Nullable` previously required explicit unwrapping, which ensured null values were caught early. Anyway it's internal.